tests/dummy/app/initializers/route-anchor-jump.js

```diff
@@ -0,0 +1,32 @@
+import Route from '@ember/routing/route';
+import { schedule } from '@ember/runloop';
+
+/**
+ * This service is a temporary fix to unblock this repository from an
```

Comment:
Minor nit:

```suggestion
 * This initializer is a temporary fix to unblock this repository from an
```

Also, would you mind making an issue for removing this file once a ember-cli-addon-docs release is done?

Reply:
Done. Will create an issue.